src/Orbit/Models/ThermalSystem.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Orbit.Models
+{
+    class ThermalSystem
+    {
+        double actualCabinTemp;
+        double selectedCabinTemp;
+
+        double internalCoolantTemp;
+        bool internalCoolantPumpOn;
+        double internalCoolantPumpTemp;
+        double internalCoolantPumpVibration;
+        int internalCoolantTankLevel;
+        bool cabinCoolantLeakSensorTripped;
+
+        double externalCoolantTemp;
+        bool externalcoolantPumpOn;
+        double externalCoolantPumpTemp;
+        double externalCoolantPumpVibration;
+        int externalCoolantTankLevel;
```

Comment:
Would it make sense to encapsulate these into another object too?

Reply:
These were moved into another class as well, ExternalCoolantLoop.cs